Add CSS style to the website

JulioOrdonezV opened the pull request:
Added CSS style to the website using Bootstrap solid template. Removed unnecessary files. fixes #5

JulioOrdonezV committed:
Add CSS style to the website

Added CSS style to the website using Bootstrap solid template. Removed unnecessary files.

cdeliens commented on app/views/pages/convert_index.html.erb:
```diff
@@ -1,2 +0,0 @@
-<h1>Hello, Rails!</h1>
```
?

JulioOrdonezV replied:
just some junk that was laying around :smile:

cdeliens commented:
All assets that are from 3rd parties should be moved under "vendor/assets", I saw a lot of them under "app/assets" even under "public" folder. In production Rails will precompile the assets and move them to the public folder so I would prefer to handle then also in the vendor/assets folder you can create a "fonts" folder if needed

JulioOrdonezV committed:
remove bootstrap gem

JulioOrdonezV committed:
Move stylesheets and javascripts to the vendor folder

Moved all the css and js files to the vendor folder, updated application.js and application.css to import those files from the vendor folder.

JulioOrdonezV committed:
Add saas gem it was deleted by mistake
